src/client/main.tsx

```diff
@@ -15,7 +21,10 @@ export class Main extends React.Component<{}, ClientState> {
 
   constructor() {
     super();
-    this.state = { stage: Stages.SPLASH };
+    this.state = {
+      stage: Stages.SPLASH,
+      numPlayersInLobby: 0,
```

Comment:
Hm actually I don't like how we have a `lobby` thing in `Main`.  Not worth the time effort atm, but maybe we should do a refactor pass after this deliverable to make the architecture nicer before the full deliverable 4